lib/lookup.json

```
@@ -420,6 +420,7 @@
     "comment": "postlint script fails"
   },
   "serialport": {
+    "skip": ["aix", "s390x", "rhel"],
```

I *think* this will work:

```suggestion
    "skip": ["ppc", "s390x"],
```

AIX in CI is ppc. rhel will skip rhel-x64 which is passing. I *think* we just want to skip ppc and s390x. (Skipping on processor type probably makes more sense here anyway, since it's a can't-compile-for-that-processor issue, I believe.)

---

Thank you. Seeing both "aix", and "ppc" elsewhere in the lookup was a little confusing.